pkg/test/dnszone/dnszone.go

```diff
@@ -1,11 +1,10 @@
 package dnszone
 
 import (
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
 	"github.com/openshift/hive/pkg/constants"
 	"github.com/openshift/hive/pkg/test/generic"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
This import should stay where it was. The k8s imports should come in a section before the hive imports.

Reply:
I think my editor probably did that (probably because I removed a line that used it so it deleted it, and then I put the line back and it re-added it). At any rate, this wasn't an intentional change. Will fix.